velox/functions/prestosql/aggregates/tests/ReduceAggTest.cpp

```diff
@@ -382,9 +346,7 @@ TEST_F(ReduceAggTest, fuzzGlobalAvg) {
        "(s, x) -> (row_constructor(s.sum + x, s.count + 1)), "
        "(s, s2) -> (row_constructor(s.sum + s2.sum, s.count + s2.count)))"},
       {"a0.sum / cast(a0.count as double)"},
-      [&](auto& builder) { return builder.assertResults(avgResults); },
-      /*config*/ {},
-      /*testWithTableScan*/ false);
+      [&](auto& builder) { return builder.assertResults(avgResults); });
```

Comment:
Is this change sufficient to reproduce the bug? If so, maybe remove lazyConstantInput test.

Reply:
Yes, existing tests are sufficient to reproduce the bug when testWithTableScan is true. I removed the lazyConstantInput test.